netskope_connector.py

```diff
@@ -1742,27 +1741,27 @@ def _handle_update_url_list(self, param):
         """
         self.save_progress(('In action handler for: {0}').format(self.get_action_identifier()))
         action_result = self.add_action_result(ActionResult(dict(param)))
-        ret_val = self._update_url_helper(action_result, "update_url")
+        ret_val = self._update_url_helper(action_result)
         if phantom.is_fail(ret_val):
             return action_result.get_status()
         return action_result.set_status(phantom.APP_SUCCESS)
 
-    def _update_url_helper(self, action_result, action_name=None):
+    def _update_url_helper(self, action_result):
         """ Helper function for updating URL."""
         try:
-            config = self.get_config()
             exists, message, content = self.get_url_list()
+            content = [element.strip() for element in content if element.strip()]
+            if not content:
+                return action_result.set_status(phantom.APP_ERROR, "No content found to update the url list")
             params = {'list': (',').join(content), 'name': self._list_name}
-            if(action_name == "update_url"):
-                params.update({'name': self._unicode_string_handler(config[NETSKOPE_LIST_NAME])})
             self._log.info(('action=get_url_list exists={} message={} content_length={}').format(exists, message, len(content)))
             request_status, request_response = self._make_rest_call(endpoint=NETSKOPE_URL_LIST_ENDPOINT,
                     action_result=action_result, params=params)
             if phantom.is_fail(request_status):
                 return action_result.get_status()
             action_result.add_data({})
             summary = action_result.update_summary({})
-            summary['total_files'] = action_result.get_data_size()
```

Comment:
Do we need to remove this field? 
This can break the playbooks of other customers who are using this field in their playbooks.

Reply:
The 'total_files' summary key was unnecessary and did not provide any useful information to the users. We have replaced it with an equivalent summary key of 'total_urls' which will display the total entities updated on Netskope UI. 
For backward compatibility, we have added this point in the readme.